### ISSUE TITLE
Plex not start

### ISSUE THREAD

**Issue by louia** [linuxserver/docker-plex#329]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior

Start well

## Current Behavior

Critical error at startup
`Critical: libusb_init failed`
## Steps to Reproduce

Docker compose : 
```yml
  plex:
    image: ghcr.io/linuxserver/plex
    container_name: plex
    restart: unless-stopped
    network_mode: host
    environment:
      - PUID=${PUID}
      - PGID=${PGID}
      - TZ=Europe/Paris
      - PLEX_CLAIM=XXX
    volumes:
      - ${PATH_FOLDER}/plex:/config
      - ${PATH_MEDIA}:/data
 ```

## Environment
Machine : Pi 4 8gb
OS: debian 11 bulleye latest
All package updated

Docker version : 20.10.17
Docker compose version : 2.2.3
**CPU architecture:** arm64
**How docker service was installed:**

## Command used to create docker container (run/create/compose/screenshot)

docker-compose up -d plex

## Docker logs

```
plex  |
plex  | -------------------------------------
plex  |           _         ()
plex  |          | |  ___   _    __
plex  |          | | / __| | |  /  \
plex  |          | | \__ \ | | | () |
plex  |          |_| |___/ |_|  \__/
plex  |
plex  |
plex  | Brought to you by linuxserver.io
plex  | -------------------------------------
plex  |
plex  | To support LSIO projects visit:
plex  | https://www.linuxserver.io/donate/
plex  | -------------------------------------
plex  | GID/UID
plex  | -------------------------------------
plex  |
plex  | User uid:    0
plex  | User gid:    0
plex  | -------------------------------------
plex  |
plex  | cont-init: info: /etc/cont-init.d/10-adduser exited 0
plex  | cont-init: info: running /etc/cont-init.d/40-chown-files
plex  | cont-init: info: /etc/cont-init.d/40-chown-files exited 0
plex  | cont-init: info: running /etc/cont-init.d/45-plex-claim
plex  | cont-init: info: /etc/cont-init.d/45-plex-claim exited 0
plex  | cont-init: info: running /etc/cont-init.d/50-gid-video
plex  | cont-init: info: /etc/cont-init.d/50-gid-video exited 0
plex  | cont-init: info: running /etc/cont-init.d/60-plex-update
plex  |
plex  |
plex  |
plex  | #######################################################
plex  | # Update routine will not run because you havent set  #
plex  | # the VERSION variable or you opted out of updates.   #
plex  | #          For more information checkout :-           #
plex  | #    https://github.com/linuxserver/docker-plex       #
plex  | #######################################################
plex  |
plex  |
plex  | cont-init: info: /etc/cont-init.d/60-plex-update exited 0
plex  | cont-init: info: running /etc/cont-init.d/90-custom-folders
plex  | cont-init: info: /etc/cont-init.d/90-custom-folders exited 0
plex  | cont-init: info: running /etc/cont-init.d/99-custom-scripts
plex  | [custom-init] no custom files found exiting...
plex  | cont-init: info: /etc/cont-init.d/99-custom-scripts exited 0
plex  | s6-rc: info: service legacy-cont-init successfully started
plex  | s6-rc: info: service init-mods: starting
plex  | s6-rc: info: service init-mods successfully started
plex  | s6-rc: info: service init-mods-package-install: starting
plex  | s6-rc: info: service init-mods-package-install successfully started
plex  | s6-rc: info: service init-mods-end: starting
plex  | s6-rc: info: service init-mods-end successfully started
plex  | s6-rc: info: service init-services: starting
plex  | s6-rc: info: service init-services successfully started
plex  | s6-rc: info: service legacy-services: starting
plex  | services-up: info: copying legacy longrun plex (no readiness notification)
plex  | s6-rc: info: service legacy-services successfully started
plex  | s6-rc: info: service 99-ci-service-check: starting
plex  | [ls.io-init] done.
plex  | s6-rc: info: service 99-ci-service-check successfully started
plex  | Starting Plex Media Server.
plex  | Critical: libusb_init failed
```

An no port expose by plex by running `docker-compose ps` : 
![image](https://user-images.githubusercontent.com/33846211/187021226-71f7871e-d524-411a-8413-22fc7d2ba266.png)

**Comment by j0nnymoe** [comment]
This error does not stop Plex from starting (if you'd checked previous issues) I suspect part of the problem is the fact your container is starting at root (0) for some reason.

**Comment by hadim** [comment]
I am having the same issue while my setup hasn't changed.

**Comment by j0nnymoe** [comment]
@hadim this is not helpful.

**Comment by hadim** [comment]
Ok after some investigations, that was due to the recent Plex breach after resetting my password. The process seems to not be very well-supported by Plex when it comes to reset the bridge in between your server and account. So you must reinitialize it by manually modifying the config of your server. You can do that following https://support.plex.tv/articles/204281528-why-am-i-locked-out-of-server-settings-and-how-do-i-get-in/

In short: open `Preferences.xml` and remove the following entries: 

PlexOnlineHome="1"
PlexOnlineMail="jane@example.com"
PlexOnlineToken="RanDoMHexIDecIALtoKeNheRE"
PlexOnlineUsername="ExampleUser"

Pretty sure, a lot of other users will have the same issue so I hope it helps.

**Comment by aptalca** [comment]
Here's our specific instructions: https://info.linuxserver.io/issues/2022-08-24-plex/

**Comment by hadim** [comment]
Thanks. I would suggest adding it to the README for a few weeks/months. I missed that blog post and would guess others might miss it as well.

**Comment by aptalca** [comment]
That site is where all our important notifications are posted. We're not gonna edit the readmes and push new builds every time there is a notice. You can subscribe to whichever topics or containers you're interested in.
https://info.linuxserver.io/
